### PR TITLE
Issues misconfigured

### PR DESCRIPTION
**Pull request by wadoon**
Due to missing issue templates and misconfiguration, nobody can currently create an issue. 

This commit fixes this, and also repairs the link to the FAQs.

### CODE REVIEW

**Comment by mattulbrich** [comment]
Hang on.

I think it would be a very good idea that we use templates instead of free text for bug reports. @WolframPfeifer said he would look into this soon and provide templates (or even forms)

**Comment by mi-ki** [comment]
I fully agree with @mattulbrich on this point. Starting the new process with (allowing for) chaotic issue reporting does not appear sensible to me. Hence, this is not a misconfiguration in my eyes.

**Comment by wadoon** [comment]
> Hang on.
> 
> I think it would be a very good idea that we use templates instead of free text for bug reports. @WolframPfeifer said he would look into this soon and provide templates (or even forms)

The creation of blank issues is not that bad. I am not aware when #3004 will be ready, but currently it is not possible to create an issue! And until #3004, we should just allow it.

The difference is a small hint at the end of the list, that blank issues are allowed, [see here](https://github.com/wadoon/key-playground/issues/new/choose). 

On the other side, "issue templates" do not guarantee anything. The user can delete everything in the issue and just start writing own unstructured reports.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
-blank_issues_enabled: false
+blank_issues_enabled: true
```
Comment (mi-ki) [inline .github/ISSUE_TEMPLATE/config.yml:1]: Blank issues are not a sensible idea for a structured development process.

**Review by mi-ki — CHANGES_REQUESTED** [review]
The proposed pull request is neither a fix nor an improvement. Instead, e.g., issue forms with mandatory fields should be used.

**Comment by mattulbrich** [comment]
I've made a suggestion to #3004 and think we can merge that now ... it'll do for the moment. ...

**Pull request closed without merging**